### ISSUE TITLE
budgeting freetext filter: acessibility/ screenreader

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4569]
**URL:** https://meinberlin-dev.liqd.net/projekte/burgerhaushalt-3-phasen-testprojekt/?mode=list
**user:** any
**expected behaviour:** screenreader reads out the number of search results
**behaviour:** screenreader does not read out the number of search results
**important screensize:**
**device & browser:** 
**Comment/Question:** I hope I tested correctly, please check again.

Screenshot?

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer we are still blocked here so will remove from milestone

**Comment by phillimorland** [comment]
@CarolingerSeilchenspringer still blocked here, shall I remove it from the milestone?

**Comment by CarolingerSeilchenspringer** [comment]
@phillimorland @fuzzylogic2000 I handed over this issue to @jostertag who will have a look next week.

**Comment by SandisStudi** [comment]
NVDA (Firefox) and JAWS (Chrome) is reading the number of search results when reading through.

https://www.accessibility-developer-guide.com/knowledge/screen-readers/desktop/reading-websites/
This was really helpful! Thanks @phillimorland

**Comment by jostertag** [comment]
@CarolingerSeilchenspringer @phillimorland Voice Over (Mac) is also reading the number of search results when reading through.

**Comment by fuzzylogic2000** [comment]
So, we can close this issue, right? Reopen if that's not the case.
